test/integration/README

```diff
@@ -0,0 +1,54 @@
+# dbus-broker integration test suite
+
+dbus-broker's integration test suite uses tmt (Test Management Tool [0]) to organize and run tests. Since tmt
+offers a _lot_ of features, this document pinpoints the most _interesting_ ones to get stuff up and running
```

Comment:
@mrc0mmand just out of curiosity I wonder if it's possible to build things with ASan/UBSan and install that to the test environment somehow? Valgrind would be nice too.

(As far as I can remember it came up recently elsewhere in the context of moving stuff like that from GitHub Actions to the Packit infrastructure)

Reply:
That's definitely on my TODO list (i.e. running dbus-broker with ASan/UBSan). I'm not sure how I'll do that, but that's something for my future me to figure out :)